products/big-ip.md

```diff
@@ -0,0 +1,44 @@
+---
+title: BIG-IP 
+category: os
+permalink: /big-ip
+versionCommand: show /sys version
+releaseDateColumn: true
+eolColumn: End of Technical Support
+eoasColumn: End of Software Development
+
+releases:
+
+-   releaseCycle: "17.1.x"
+    releaseDate: 2023-03-14
+    eoas: 2027-03-31
+    eol: 2027-03-31
+    latest: "17.1.1"  
+    lts: true
+
+-   releaseCycle: "16.1.x"
+    releaseDate: 2021-07-07
+    eoas: 2025-07-31
+    eol: 2025-07-31
+    latest: "16.1.5"
+    lts: true
+
+-   releaseCycle: "15.1.x"
+    releaseDate: 2019-12-11
+    eoas: 2025-12-31
+    eol: 2025-12-31
+    latest: "15.1.10"
+    lts: true
+
+---
+
+> [BIG-IP](https://www.f5.com/products/big-ip) is a family of hardware and software products from F5 Networks, designed for load balancing, traffic management, and application security.
+
+
+**Summary of the release policy**:  
+   F5 Networks typically provides a major release of BIG-IP every 12-18 months, with regular maintenance updates to address security and performance issues. Each version of BIG-IP is generally supported for up to 5 years, including Full Support for the first 3 years and Limited Support for the remaining 2 years. After this period, the product reaches its End of Life (EoL) and will no longer receive updates or support from F5. The **EoL policy** ensures that customers have a defined timeline for transitioning to newer versions before support fully ends.
```

Comment:
```suggestion
F5 Networks typically provides a major release of BIG-IP every 12-18 months, with regular maintenance updates to address security and performance issues. Each version of BIG-IP is generally supported for up to 5 years, including Full Support for the first 3 years and Limited Support for the remaining 2 years.
```

The two last sentences don't add much value IMO and could be removed. WDYT ?

Moreover the paragraph states "up to 5 years" of support. From what I understood by quickly reading https://my.f5.com/manage/s/article/K5903, it is systematically 4 years from 16.1. The description should describe the current policy. Older policy may be mentioned too as a side note.